geopyspark-backend/geotrellis/src/main/scala/geopyspark/geotrellis/SpatialTiledRasterLayer.scala

```diff
@@ -192,36 +192,36 @@ class SpatialTiledRasterLayer(
     PythonTranslator.toPython[Tile, ProtoTile](contextRDD.stitch.tile)
   }
 
-  def save_stitched(path: String): Unit =
-    _save_stitched(path, None, None)
-
-  def save_stitched(path: String, cropBounds: ArrayList[Double]): Unit =
-    _save_stitched(path, Some(cropBounds), None)
-
-  def save_stitched(path: String, cropBounds: ArrayList[Double], cropDimensions: ArrayList[Int]): Unit =
-    _save_stitched(path, Some(cropBounds), Some(cropDimensions))
-
-  private def _save_stitched(path: String, cropBounds: Option[ArrayList[Double]], cropDimensions: Option[ArrayList[Int]]): Unit = {
-
+  def saveStitched(path: String): Unit =
+    saveStitched(path, None, None)
+
+  def saveStitched(path: String, cropBounds: java.util.Map[String, Double]): Unit =
+    saveStitched(path, Some(cropBounds), None)
+
+  def saveStitched(
+    path: String,
+    cropBounds: java.util.Map[String, Double],
+    cropDimensions: ArrayList[Int]
+  ): Unit =
+    saveStitched(path, Some(cropBounds), Some(cropDimensions))
```

Comment:
Was mostly pointing out a pattern that I think is good practice. Its definitely guarded in this case but  potentially this may have another call origin that won't do the same check or some other tomfoolery.